tools/gulp/docs/processMarkdownPage.js

```diff
@@ -51,9 +51,11 @@ function processMarkdownPage(filePath, body, rootPath = '') {
     referenceURI = path.join(rootPath, referenceURI);
   }
 
+  const header = parts.attributes.title || 'Untitled';
   let page = {
     depth: depth,
-    header: parts.attributes.title || 'Untitled',
+    label: parts.attributes.label || header,
```

Comment:
`label` is the new optional front matter argument that allows the nav label to be different than the actual page title. I couldn't figure out how to add this support to KSS, so I only added the functionality to the markdown files. I'm hoping we can eventually move away from KSS altogether so this won't matter. 

At the very least, now its consistent where only the component & . utilities source pages use KSS, and everything else is markdown.